src/shared/components/query/QueryStore.ts

```diff
@@ -296,13 +225,21 @@ export class QueryStore
 	@computed get selectedStudyIds():string[]
 	{
 		let ids:string[] = this._selectedStudyIds.keys();
-		const selectableStudies = this.selectableStudiesSet;
-		ids = ids.filter(id=>!!selectableStudies[id]);
+		const selectableStudies = this.selectableIdToPhysicalIdsMap.result;
+		ids = ids.reduce((obj:string[],next)=>{
+			if(selectableStudies[next]){
+				return obj.concat(selectableStudies[next])
+			}
+			return obj;
+
+		},[]);
 		return this.forDownloadTab ? ids.slice(-1) : ids;
 	}
 
 	set selectedStudyIds(val:string[]) {
-		this._selectedStudyIds = observable.map(stringListToSet(val));
+		//filter out deleted virtual study

```

Comment:
yeah, i don't think this is right.  we shouldn't need to filter out deleted studies at this point.  is there a reason we need to keep track of deleted studies?  (as in, we need to display them in some deleted list).

Reply:
action(()=>)